src/main/java/org/jenkinsci/plugins/ccxml/CCXMLAction.java

```diff
@@ -51,14 +62,45 @@ public String getDisplayName() {
 
     @Override
     public String getUrlName() {
-        return "cc.xml2";
+        return URL_NAME;
     }
 
     public View getView() {
         return this.view;
     }
 
     /**
+     * @return A map containing the items in the view object. If the request
+     * contains a query parameter named "recursive", then folders in the view
+     * are traversed recursively and all items in those folders are returned
+     * as well. The map is keyed by the folder the item is in, with top-level
+     * items having an empty key.
+     */
+    public Map<String, Collection<TopLevelItem>> getItems() {
```

Comment:
If it is considered as a backportable bugfix, new `public` methods should be marked as `@Restricted(NoExternalUse.class)`